packages/react-app/src/lib/bridge.js

```diff
@@ -72,6 +80,24 @@ const fetchToTokenDetails = async (bridgeDirection, fromToken, toChainId) => {
     toChainId,
   );
 
+  if (NATIVE_CURRENCY_SYBMOLS.includes(fromSybmol)) {
```

Comment:
There might be token BNB on mainnet or ETH or BSC or something else like that. Should do this in a cleaner manner.